src/test/java/core/basesyntax/store/StorageServiceImplTest.java

```diff
@@ -0,0 +1,67 @@
+package core.basesyntax.store;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImpl;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitRecord;
+import core.basesyntax.model.OperationType;
+import core.basesyntax.store.strategy.BalanceHandler;
+import core.basesyntax.store.strategy.PurchaseHandler;
+import core.basesyntax.store.strategy.ReturnHandler;
+import core.basesyntax.store.strategy.SupplyHandler;
+import core.basesyntax.store.strategy.TypeHandler;
+import core.basesyntax.validator.quantity.QuantityValidator;
+import core.basesyntax.validator.quantity.QuantityValidatorImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StorageServiceImplTest {
+    private static List<FruitRecord> fruitRecordList;
+    private static StorageService storageService;
+    private static final String APPLE = "apple";
+    private static final long QUANTITY = 100;
+    private static Fruit fruit;
+    private static final OperationType BALANCE = OperationType.B;
+
+    @BeforeClass
+    public static void setUp() {
+        Map<OperationType, TypeHandler> typeHandlerMap = new HashMap<>();
+        FruitDao fruitDao = new FruitDaoImpl();
+        QuantityValidator quantityValidator = new QuantityValidatorImpl();
```

Comment:
we can have private methods in tests, FYI

Reply:
Can you say how, because of the compiler throws an error

Reply:
![image](https://user-images.githubusercontent.com/58442482/117293336-e9b28a00-ae79-11eb-883c-3a68e2a5ddaa.png)

Reply:
like an example